ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/geocoder.js

```diff
@@ -66,38 +66,44 @@ module.exports = Backbone.Model.extend({
   },
   queryGazetteerOffline(input) {
     // Load in request string
-    return Cesium.loadText(geocoderOfflineEndpoint + input)
-      .then(function(results) {
-        // Strip out unwanted characters and parse JSON
-        var jsonResult = /\((.+)\)/.exec(results)[1]
-        var jsonObject = JSON.parse('[' + jsonResult + ']')
-
-        // Extract desired fields
-        return jsonObject.map(function(locationResult) {
-          var resultValues = locationResult.resourceSets[0].resources[0]
-          if (typeof resultValues != 'undefined') {
-            return {
-              displayName: resultValues.name,
-              destination: Cesium.Rectangle.fromDegrees(
-                resultValues.bbox[1],
-                resultValues.bbox[0],
-                resultValues.bbox[3],
-                resultValues.bbox[2]
-              ),
+    return new Promise((resolve, reject) => {
+      $.ajax({
+        url: geocoderOfflineEndpoint,
+        data: 'jsonp=jsonp&query=' + input,
+        contentType: 'application/javascript',
+        dataType: 'jsonp',
+        jsonp: 'jsonp',
+        success: function (result) {  
+          // Extract desired fields
+          result = [result]
+          resolve(result.map(function (locationResult) {
+            var resultValues = locationResult.resourceSets[0].resources[0]
+            if (typeof resultValues != 'undefined') {
```

Comment:
a ternary might be more readable here

Reply:
:exclamation: Equality is usually more readable than in-equality, and **it should be `!==` or `===`**.